.travis.yml

```diff
@@ -35,8 +52,8 @@ addons:
 
 install:
   - |
-    if [ $TRAVIS_PHP_VERSION != '5.6' ]; then
-        # disable xdebug for performance reasons when code coverage is not needed
+    if [[ $TRAVIS_PHP_VERSION != '5.6' || $TRAVIS_PHP_VERSION != hhv* ]]; then
```

Comment:
this needs to be `&&`, not `||`. We want to disable xdebug on all php versions that are not 5.6 **and** not hhvm.